assets/xml/objects/object_bee.xml

```diff
@@ -1,28 +1,28 @@
 ﻿<Root>
     <File Name="object_bee" Segment="6">
```

Comment:
Can you also add small comment between the `Root` and `File` node describing what the asset is for. See `object_giant.xml` for an example

Reply:
fixed in 75be92a 👍